Tour of Heroes/tests/test.js

```diff
@@ -1,3 +1,4 @@
+//comment for pull req
```

Comment:
missing tests for:
find user by id when user doesnt exist
add new user when user already exists
update user when user doesnt exist
delete user (id or by name) when doesnt exist

except for that, best tests ive seen so far.
good job!!